runbook/README.md

```diff
@@ -0,0 +1,106 @@
+on AWS are identical, and I will explicitly call out the places where
+different steps are required. Throughout the document, we assume that
+your client machine is a UNIX-like environment (e.g., OS X, Linux, or WSL).
+
+## Local Client Setup
+
+I need a few things to be true about the local machine that you will be using
+to connect to AWS that I'll outline here:
+
+1. You have created and downloaded a `.pem` file for connecting to an EC2
+instance to your `~/.ssh` directory. When we provision machines, you'll need to
+use the `.pem` file for connecting.
+1. You have created a `~/.ssh/config` file that contains an entry that looks like this so we can use `staging` as an alias for your
+   provisioned EC2 instance in the rest of the runbook:
+
+    ```
+    host staging
+    HostName <IP address of provisioned server goes here>
+    IdentityFile ~/.ssh/<your .pem file goes here>
+    User ec2-user
+    IdentitiesOnly yes
+    StrictHostKeyChecking no 
+    ```
+1. You can [connect to Github via SSH.](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) This is
+important because we will need to use your Github SSH key to interact with private repositories from the `staging` server on EC2.
+
+## Provisioning The Staging Server
+
+If you are running an *Inference* job, you should
+use a small instance type for your staging server (e.g., an `m5.xlarge` will be more than enough.) If you are running a *Planning* job, you
+should provision a beefy instance type (I am especially partial to the memory-and-CPU heavy `r5.24xlarge`, but given how fast the planning code
+has become, an `r5.8xlarge` should be perfectly adequate.)
+
+If you have access to the `jh-covid` account, you should use the *IDD Staging AMI* (`ami-03641dd0c8554e5d0`) to provision and launch new
+staging servers; it is already setup with all of the dependencies described in this section. You can find the AMI [here](https://us-west-2.console.aws.amazon.com/ec2/v2/home?region=us-west-2#Images:sort=imageId), select it, and press the *Launch* button to walk you through the Launch Wizard to choose your instance type and `.pem` file to provision your staging server.
+Once your instance is provisioned, be sure to put its IP address into the `HostName` section of the `~/.ssh/config` file on your local client so that you can connect to it from your
+client by simply typing `ssh staging` in your terminal window.
+
+If you do *not* have access to the `jh-covid` account, you should walk through the regular EC2 Launch Wizard flow and be sure to choose the
+*Amazon Linux 2 AMI (HVM), SSD Volume Type* (`ami-0e34e7b9ca0ace12d`, the 64-bit x86 version) AMI. Once the machine is up and running and you can
+SSH to it, you will need to run the following code to install the software you will need for the rest of the run:
+
+```
+sudo yum -y update
+sudo yum -y install awscli 
+sudo yum -y install git 
+sudo yum -y install docker.io 
+sudo yum -y install pbzip2 
+
```

Comment:
if you add `sudo usermod -G docker` to add the current user to the docker group, you won't have to sudo docker

Reply:
Ooh, TIL!